infrastructure/templates/public-api/main.bicep

```diff
@@ -0,0 +1,362 @@
+
+@description('Database : Azure Database for PostgreSQL sku name ')
+@allowed([
+  'Standard_B1ms'
+  'Standard_D4ads_v5'
+  'Standard_E4ads_v5'
+])
+param skuName string = 'Standard_B1ms'
+
+@description('Database : Azure Database for PostgreSQL Storage Size in GB')
+param storageSizeGB int = 32
+
+@description('Database : Azure Database for PostgreSQL Autogrow setting')
+param autoGrowStatus string = 'Disabled'
+
+//Container Registry Params ----------------------------------------------------------------
+@minLength(5)
+@maxLength(50)
+@description('Registry : Name of the azure container registry (must be globally unique)')
+param containerRegistryName string = 'eesapiacr'
+
+//Container App Params
+@description('Container App : Specifies the container image to deploy from the registry <name>:<tag>.')
+param acrHostedImageName string = 'hello-world'
+
+@description('Specifies the container port.')
+param targetPort int = 80
+
+@description('Container App : Specifies the container image to seed to the ACR.')
+param containerSeedImage string = 'mcr.microsoft.com/azuredocs/aci-helloworld'
+
+@description('Select if you want to seed the ACR with a base image.')
+param seedRegistry bool = true
+
+//ServiceBus Queue Params -------------------------------------------------------------------
+@description('Name of the Service Bus namespace')
+param namespaceName string = 'etlnamespace'
+
+@description('Name of the Queue')
+param queueName string = 'etlfunctionqueue'
+
+//ETL Function Paramenters ------------------------------------------------------------------
+@description('Application : Insights name')
+param applicationInsightsName string = 'etlFunctionInsights'
+
+@description('Function App Plan operating system')
+@allowed([
+  'Windows'
+  'Linux'
+])
+param appServiceplanOS string = 'Linux'
+
+@description('Function App : Runtime Language')
+@allowed([
+  'dotnet'
+  'node'
+  'python'
+  'java'
+])
+param functionAppRuntime string = 'python'
+
+//---------------------------------------------------------------------------------------------------------------
+// All resources via modules
+//---------------------------------------------------------------------------------------------------------------
+
+//Deploy Networking
+module vnetModule 'components/network.bicep' = {
+  name: 'virtualNetworkDeploy'
+  params: {
+    subscription: subscription
+    location: location
+    environment: environment
+    deploySubnets: deploySubnets
+    //tags
+    departmentName: departmentName
+    environmentName: environmentName
+    solutionName: solutionName
+    subscriptionName: subscriptionName
+    costCentre: costCentre
+    serviceOwnerName: serviceOwnerName
+    dateProvisioned: dateProvisioned
+    createdBy: createdBy
+    deploymentRepo: deploymentRepo
+    deploymentScript: 'network.bicep'
+  }
+}
+
+//Deploy Storage Account
+module storageAccountModule 'components/storageAccount.bicep' = {
```

Comment:
Whilst the casing of the filename in this string, it's not actually right in the project files. It's currently `storageaccount.bicep`, so we should correct this